letsencrypt-auto-source/tests/centos6_tests.sh

```diff
@@ -47,6 +47,12 @@ if [ $RESULT -eq 0 ]; then
   exit 1
 fi
 
+# Skip self upgrade due to LE_PYTHON value
+if ! LE_PYTHON=nonexistant certbot/letsencrypt-auto-source/letsencrypt-auto 2>&1 | grep -q "WARNING: couldn't find Python"; then
```

Comment:
nonexistent

Reply:
unless you're worried nonexistent exists

Reply:
The purpose of this was to set `LE_PYTHON` to a nonexistent value, however, the empty string is also a good nonexistent value so I removed setting the variable.

Reply:
I read this on my phone last night, didn't notice the spelling difference, and thought you had a problem with this approach, however, I think testing with the real (empty) `LE_PYTHON` value is probably better than testing with a silly value like this so I'm going to leave this PR as is.

Reply:
Good with either test for now; if something in #5388 I'll fix the test there. Do we not care about line length in leauto?